src/Web/Promise/Lazy.purs

```diff
@@ -68,3 +68,12 @@ race :: forall a. Array (LazyPromise a) -> LazyPromise a
 race as = LazyPromise do
   as' <- traverse (\(LazyPromise a) -> a) as
   runEffectFn1 P.race as'
+
+fromPromise :: forall a. Effect (P.Promise a) -> LazyPromise a
+fromPromise p = LazyPromise $ runEffectFn2 P.then_ (mkEffectFn1 (pure <<< P.resolve <<< Box)) =<< p
+
+toPromise :: forall a. LazyPromise a -> Effect (P.Promise a)
+toPromise (LazyPromise p) = unbox =<< p
```

Comment:
These two functions could also be renamed `toLazyPromise :: Effect (Promise a) -> LazyPromise a` and `fromLazyPromise :: LazyPromise a -> Effect (Promise a)` if desired. I'm open to other names.

Reply:
I think the current names are good since they're in the `Lazy` module already - I assume the common usage would be something like: `LazyPromise.toPromise`, whereas I'd call them `Promise.toLazyPromise` etc. if they lived in the non-lazy module 🙂